app/classes/shop.py

```diff
@@ -0,0 +1,34 @@
+import datetime
+
+
+class Shop:
+    def __init__(
+            self,
+            name: str,
+            location: list,
+            products: dict
+    ) -> None:
+        self.name = name
+        self.location = location
+        self.products = products
+
+    def print_receipt(
+            self,
+            customer_name: str,
+            customer_cart: dict
+    ) -> str:
+        date = datetime.datetime.now().strftime("%d/%m/%Y %H:%M:%S")
+        receipt = (f"Date: {date}\n"
+                   f"Thanks, {customer_name}, for your purchase!\n"
+                   f"You have bought:\n")
```

Comment:
Try to use same styling. I meant multi-lined string in parenthesis same way, as your init and print_receipt args. Aligning the opening and closing parentheses and keeping the indented lines at the same level is a good practice and makes the code more aesthetically pleasing and easier to follow.